### PR TITLE
Add tests to delete revisions for a single post

### LINKED ISSUE #252: Delete Post Revision - Add tests
## Tests
- [x] Add tests to delete revisions for a single post
- [x] Add tests to delete revisions for multiple posts

### PR DESCRIPTION
**Pull request by rajanit2000**
Add tests to delete revisions for multiple posts

Fix #252

### CODE REVIEW

File: tests/wp-unit/include/Core/Posts/Modules/DeletePostsByRevisionModuleTest.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	public function test_delete_revisions_for_single_post() {
+		
+		$post = $this->factory->post->create( array(
+			'post_type' => 'post',
+		) );
+
+		$revision_post_1 = array(
+			'ID'           => $post,
+			'post_title'   => rand(),
+			'post_content' => md5( rand() ),
```
Comment (sudar) [inline tests/wp-unit/include/Core/Posts/Modules/DeletePostsByRevisionModuleTest.php]: @rajanit2000 

This is a clever way to update the post. Good job.

File: tests/wp-unit/include/Core/Posts/Modules/DeletePostsByRevisionModuleTest.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+		$this->module = new DeletePostsByRevisionModule();
+	}
+
+	/**
+	 * Test delete revisions for a single post
+	 */
+	public function test_delete_revisions_for_single_post() {
+		
+		$post = $this->factory->post->create( array(
```
Comment (sudar) [inline tests/wp-unit/include/Core/Posts/Modules/DeletePostsByRevisionModuleTest.php]: Rename this variable to `post_id`

File: tests/wp-unit/include/Core/Posts/Modules/DeletePostsByRevisionModuleTest.php [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		$this->assertEquals( 2, $posts_deleted );
+
+	}
+
+	/**
+	 * Test delete revisions for a multiple post
+	 */
+	public function test_delete_revisions_for_multiple_post() {
+		
+		$posts = $this->factory->post->create_many( 10, array(
```
Comment (sudar) [inline tests/wp-unit/include/Core/Posts/Modules/DeletePostsByRevisionModuleTest.php]: Rename this variable to `$post_ids`

Comment (sudar) [reply tests/wp-unit/include/Core/Posts/Modules/DeletePostsByRevisionModuleTest.php]: @rajanit2000 

This is not done yet.

**Comment by rajanit2000** [comment]
Done @sudar

**Pull request merged**